src/Service.Tests/SqlTests/RestApiTests/Put/PutApiTestBase.cs

```diff
@@ -829,6 +829,87 @@ public virtual async Task PutOneWithNonNullableFieldAsNull()
                 isExpectedErrorMsgSubstr: isExpectedErrorMsgSubstr
             );
         }
+
+        /// <summary>
+        /// Test to validate failure of PUT operation failing to satisfy the database policy for the operation to be executed
+        /// (insert/update based on whether a record exists for given PK).
+        /// </summary>
+        [TestMethod]
+        public virtual async Task PutOneWithUnsatisfiedDatabasePolicy()
```

Comment:
Any success test cases for Put?

Reply:
Put/Patch have exactly the same functionality, so I thought just writing the test for Patch is sufficient. Do you think we should have them at both the places?

Reply:
Yeah I mean I know they are functionally different, I mean the same functionality in terms of what this test is trying to verify. Anyways, I will add tests for PUT as well. Thanks!

Reply:
Added tests.